src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -120,8 +120,11 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
   private def startRunnableCalls(): State = {
     tryStartingRunnableCalls() match {
       case Success(entries) =>
-        if (entries.nonEmpty) tryStartingRunnableCalls()
-        goto(WorkflowRunning)
+        if (entries.nonEmpty) {
+          startRunnableCalls()
+        } else {
+          goto(WorkflowRunning)
```

Comment:
I believe it will... just on the recursive call... I think

Honestly I leaned a lot on the tests passing to validate correctness